Frameworks/CoreGraphics/CGContext.mm

```diff
@@ -1646,7 +1647,8 @@ void CGContextFillRects(CGContextRef context, const CGRect* rects, size_t count)
 */
 void CGContextDrawPath(CGContextRef context, CGPathDrawingMode mode) {
     NOISY_RETURN_IF_NULL(context);
-    UNIMPLEMENTED();
+
+    __CGContextDrawGeometry(context, _getAndClosePathGeometry(context->Path()), mode);
```

Comment:
I made a note of this. We'll explore this more with the affine transform work item.

---
In reply to: [86265029](https://github.com/Microsoft/WinObjC/pull/1307#discussion_r86265029) [](ancestors = 86265029)